mod/Domestication/common/traits/floret_traits_admiral.txt

```diff
@@ -538,6 +538,7 @@ leader_trait_great_khan_floret = {
 
 leader_trait_militarist_1_floret = {
 	is_councilor_trait_for_ruler = yes
+	destiny_trait = yes
```

Comment:
Why add the destine_trait flag here?  I don't think it makes sense for this or the other traits you added it to.

Reply:
marking it as a destiny_trait is required if you want to use the `destiny_background_icon` that you have specified here. If this is not intended to be a destiny trait, then both should be removed.

Reply:
Fair enough.  Should be fine, then.